backend/README.md

```diff
@@ -61,12 +61,12 @@ Install the `dotnet-ef` CLI
 
 > `dotnet tool install --global dotnet-ef`
 
-Create a `connectionstrings.json` configuration file. You can also create one for each environment by creating a file with the naming conventsion `connectionstrings.[environment].json`. Enter the following information into the file;
+Create a `connectionstrings.json` configuration file within the /dal project. You can also create one for each environment by creating a file with the naming conventsion `connectionstrings.[environment].json`. Enter the following information into the file;
```

Comment:
This is optional depending on whether you want to use the `.env` file.  And is only required if you use the `dotnet ef database update`.